core/issuers.js

```diff
@@ -15,6 +15,35 @@ function makeMint(
   insist(description)`\
 Description must be truthy: ${description}`;
 
+  // The payment object is created in multiple functions (takePayment,
+  // split, combine), so we extract its particulars here and call this
```

Comment:
Would you mind adding that the payment's amount is recorded in controller/keeper, and not known to the payment?

Reply:
Yup, that sounds good